pyproject.toml

```diff
@@ -20,3 +20,96 @@ include_trailing_comma = true
 force_grid_wrap = 0
 use_parentheses = true
 line_length = 88
+
+
+[tool.poetry]
+name = "Bootcamps"
+version = "0.108.0"
+description = "Bootcamps Ecommerce"
+license = "BSD-3"
+readme = "README.md"
+packages = []
+authors  = ["MIT ODL"]
+
+[tool.poetry.dependencies]
+python = "3.9.13"
+django = "^3.2.16"
+celery = {extras = ["redis"], version = "^5.2.7"}
+boto3 = "^1.16.63"
+dj-database-url = "^0.3.0"
+django-anymail = {extras = ["mailgun"], version = "^8.4"}
+dj-static = "^0.0.6"
+django-cache-memoize = "^0.1.7"
+django-classy-tags = "^1.0.0"
+django-compat = "^1.0.15"
+django-hijack = "^2.1.10"
+django-hijack-admin = "^2.1.7"
+django-filter = "^2.4.0"
+django-fsm = "^2.8.0"
+django-ipware = "^4.0.0"
+django-mathfilters = "^1.0.0"
+django-redis = "^5.0.0"
+django-server-status = "^0.7.3"
+django-webpack-loader = "^0.7.0"
+djangorestframework = "^3.11.2"
+djangorestframework-serializer-extensions = "^2.0.0"
+djangosaml2idp = { git = "https://github.com/OTA-Insight/djangosaml2idp.git", rev = "0b4325782a6fd2c034677b5923041b5df10087ec" }
+djoser = "^2.1.0"
+dynamic-rest = "^2.1.2"
+html5lib = "^1.1"
+hubspot-api-client = "^6.1.0"
+django-storages = "^1.9.1"
+ipython = "^7.31.1"
+mitol-django-common = "^2.7.0"
+mitol-django-hubspot-api = "2023.5.22"
+mitol-django-mail= "^3.3.0"
+mitol-django-authentication = "^1.6.0"
+newrelic = "^6.2.0.156"
+premailer = "^3.7.0"
+psycopg2 = "^2.8.6"
+pycountry = "^19.8.18"
+PyNaCl = "^1.5.0"
+python-dateutil = "^2.8.1"
+pytz = "^2022.6"
+requests= "^2.20"
+sentry-sdk = "^0.14.3"
+social-auth-app-django = "^4.0.0"
+static3 = "^0.5.1"
+urllib3 = "^1.24.2"
+uwsgi = "^2.0.20"
+wagtail = "^2.15.5"
+zeep = "^3.4.0"
+Pillow = "^9.3.0"
+user-util = "^0.1.5"
+
+
+[tool.poetry.group.dev.dependencies]
+astroid = "^2.4.2"
+black = "^22.10.0"
+bpython = "^0.19"
+django-debug-toolbar = "^3.2.4"
+hypothesis = "~4.23.9"
+ipdb = "^0.13.2"
+isort = "^4.3.21"
+factory_boy = "^3.2.0"
+faker = "^8.8.2"
+nplusone = "^1.0.0"
+pdbpp = "^0.10.2"
+pip-tools = "^7.1.0"
+pylint = "~2.5.3"
+pylint-django = "~2.1.0"
```

Comment:
Using only patch version upgrade because the minor version upgrades requires changes in files as many as ~80 and doing that would increase the testing scope of this PR huge.